dashboards/mobile/mobile.json

```diff
@@ -0,0 +1,308 @@
+              ],
+              "metrics": [
+                {
+                  "name": "Mobile/Activity/Name/External/connect-dre.dbankcloud.cn",
+                  "scope": "",
+                  "values": [
+                    "call_count"
+                  ]
+                },
+                {
+                  "name": "Mobile/Activity/Name/Display HwIdSignInHubActivity",
+                  "scope": "",
+                  "values": [
+                    "call_count"
+                  ]
+                },
+                {
+                  "name": "Mobile/Activity/Name/Display InterestDetailActivity",
+                  "scope": "",
+                  "values": [
+                    "call_count"
+                  ]
+                },
+                {
+                  "name": "Mobile/Activity/Name/Display LoginActivity",
+                  "scope": "",
+                  "values": [
+                    "call_count"
+                  ]
+                },
+                {
+                  "name": "Mobile/Activity/Name/Display MainActivity",
+                  "scope": "",
+                  "values": [
+                    "call_count"
+                  ]
+                }
+              ],
+              "rawMetricName": "Mobile/Activity/Name/External/connect-dre.dbankcloud.cn"
+            }
+          },
+          {
+            "title": "OS Version",
+            "layout": {
+              "column": 9,
+              "row": 4,
+              "width": 4,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.pie"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": true
+              },
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "SELECT uniqueCount(osVersion) FROM MobileRequest FACET osVersion SINCE 1 month ago EXTRAPOLATE"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              }
+            }
+          },
+          {
+            "title": "App launches",
+            "layout": {
+              "column": 1,
+              "row": 7,
+              "width": 4,
+              "height": 3
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.billboard"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "SELECT count(newrelic.timeslice.value) AS 'App Launches' FROM Metric WHERE (entityGuid = 'MzU5NTY5NHxNT0JJTEV8QVBQTElDQVRJT058NTM1ODcyMjEx') AND (metricTimesliceName = 'Session/Start')"
```

Comment:
Entity GUIDs are unique to an account within New Relic, so if other people install this dashboard they will never get any results in this chart since their entity GUID will be different. Could you remove this WHERE clause?